### PR TITLE
Focus composer after reacting

### PR DESCRIPTION
**Pull request by SimonBrandner**
Fixes https://github.com/vector-im/element-web/issues/11023

### CODE REVIEW

**Comment by t3chguy** [comment]
Hmm won't these focus the wrong composer if used whilst editing a message?

**Comment by SimonBrandner** [comment]
> Hmm won't these focus the wrong composer if used whilst editing a message?

This is the current behaviour but we should change it

File: src/components/structures/RoomView.tsx [hunk 1]
```diff
@@ -834,17 +834,7 @@ export default class RoomView extends React.Component<IProps, IState> {
...
-                        ...payload,
-                        action: Action.FocusEditMessageComposer,
-                    });
-                } else {
-                    dis.dispatch({
-                        ...payload,
-                        action: Action.FocusSendMessageComposer,
-                    });
-                }
+                dis.fire(this.state.editState ? Action.FocusEditMessageComposer : Action.FocusSendMessageComposer);
```
Comment (t3chguy) [inline src/components/structures/RoomView.tsx:837]: could you copy this ternary to the Action.ComposerInsert re-dispatch above? waaay more readable

File: src/components/structures/LoggedInView.tsx [hunk 2]
```diff
@@ -398,7 +398,7 @@ class LoggedInView extends React.Component<IProps, IState> {
             // refocusing during a paste event will make the
             // paste end up in the newly focused element,
             // so dispatch synchronously before paste happens
-            dis.fire(Action.FocusComposer, true);
+            dis.fire(Action.FocusSendMessageComposer, true);
```
Comment (t3chguy) [inline src/components/structures/LoggedInView.tsx:401]: this one I'm not sure about, but happy to go with it for the time being

File: src/components/structures/LoggedInView.tsx [hunk 3]
```diff
@@ -552,7 +552,7 @@ class LoggedInView extends React.Component<IProps, IState> {
 
             if (!isClickShortcut && ev.key !== Key.TAB && !canElementReceiveInput(ev.target)) {
                 // synchronous dispatch so we focus before key generates input
-                dis.fire(Action.FocusComposer, true);
+                dis.fire(Action.FocusSendMessageComposer, true);
```
Comment (t3chguy) [inline src/components/structures/LoggedInView.tsx:555]: this one I'm not sure about, but happy to go with it for the time being

**Review by t3chguy — APPROVED** [review]
Great, thanks!

**Pull request merged**